helix-loader/Cargo.toml

```diff
@@ -24,7 +24,7 @@ etcetera = "0.8"
 tree-sitter.workspace = true
 once_cell = "1.19"
 log = "0.4"
-which = "5.0.0"
+which = "6.0.0"
```

Comment:
We're using `which` in several crates, maybe we should just re-export from either core or loader?